src/test/java/api/support/fakes/FakeStorageModule.java

```diff
@@ -274,17 +274,27 @@ private void getMany(RoutingContext routingContext) {
     response.end();
   }
 
-  private void empty(RoutingContext routingContext) {
+  private void deleteMany(RoutingContext routingContext) {
     WebContext context = new WebContext(routingContext);
 
-    if(!hasCollectionDelete) {
+    if (!hasCollectionDelete) {
       ClientErrorResponse.notFound(routingContext.response());
       return;
     }
 
-    Map <String, JsonObject> resourcesForTenant = getResourcesForTenant(context);
+    Integer limit = context.getIntegerParameter("limit", 1000);
```

Comment:
Is this what the real default value is?

Reply:
Why does a delete many have a page limit?

Reply:
I added this by example of `getMany()` as I haven't seen examples of "delete many" endpoint.
But I agree, it probably doesn't need limit and offset. I'll remove them.

Reply:
Does the real implementation have them?

Reply:
It doesn't

Reply:
Then definitely remove them please, these fakes are intended to mimic the real modules where possible

Reply:
Done